examples/MongoDB-Atlas-Third-Party-Integration/third-party-integration.tf

```diff
@@ -0,0 +1,15 @@
+resource "mongodbatlas_third_party_integration" "test_msteams" {
+  project_id                  = mongodbatlas_project.project.id
+  type                        = "MICROSOFT_TEAMS"
+  microsoft_teams_webhook_url = "https://mongodb0.webhook.office.com/webhookb2/zfd-15e8-47de-9a7a-355183e89a68@thi-841b-4ef9-af16-33548de0c958/IncomingWebhook/xyz"
```

Comment:
The other parts of the examples tend to use variables. It would be good to continue to follow that pattern.  

It is nice to show an example but is this actually the one I gave you or did you edit it a bit? Just want to make sure all examples are fake data.

Reply:
ty!